striptease/utilities.py

```diff
@@ -275,3 +276,74 @@ def polarimeter_iterator(
                 pol_name = BOARD_TO_W_BAND_POL[cur_board]
 
             yield cur_board, pol_idx, pol_name
+
+
+def parse_polarimeters(polarimeters: List[str]) -> List[str]:
+    """Parse a list of polarimeters, boards, "Q" (all Q polarimeters), "W" (all W polarimeters)
+    and (e.g.) "OQ", "OW" (all Q or W polarimeters in board O), and return a list of polarimeter names."""
```

Comment:
Thanks! Can you espand a bit the documentation to show the return value too? A Python snippet would be great! Something like this:

```
python> print(parse_polarimeters(["O1", "O3", "RQ", "GW"]))
["O1", "O3",  … ]
```

Reply:
Done! Let me know if I should add anything else. Thanks!